src/utils/components/AddBootableVolumeModal/utils/utils.ts

```diff
@@ -230,12 +231,28 @@ export const createDataSourceWithImportCron: CreateDataSourceWithImportCronType
   const {
     bootableVolumeName,
     cronExpression,
+    registryCredentials,
     registryURL,
     retainRevisions,
     size,
     storageClassName,
   } = bootableVolume;
 
+  const { password, username } = registryCredentials || {};
+  const addRegistrySecret = !!(username && password);
+  const imageSecretName = addRegistrySecret
+    ? `${bootableVolumeName}-registry-secret-${getRandomChars()}`
```

Comment:
seems this name can become really long. Can we cut it so that it doesn't go over 64 characters? We have a constatnt for max lenght in the name